pages/7_📋_プレイリスト履歴.py

```
@@ -139,27 +145,135 @@ def format_created_at(value: str, tz_name: str) -> str:
+                        border-left: 4px solid {'#4CAF50' if is_creator else '#2196F3'};
+                    ">
+                        <div style="font-weight: bold; margin-bottom: 4px;">
+                            {html.escape(comment_display_name)} · <span style="font-weight: normal; color: #666;">{comment_time}</span>
+                        </div>
+                        <div>{comment_display}</div>
+                    </div>
+                    """,
+                    unsafe_allow_html=True,
+                )
+
+                # 削除ボタン（自分のコメントかつ未削除の場合のみ表示）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)